src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs

```diff
@@ -522,21 +521,12 @@ public async Task ExtractManifestAsync(string nupkgPath, string targetPath)
                         throw new FileNotFoundException("Manifest MSI not found in NuGet package", extractionPath);
                     }
                     string msiExtractionPath = Path.Combine(extractionPath, "msi");
-                    
-                    var psi = new ProcessStartInfo()
-                    {
-                        FileName = "msiexec",
-                        ArgumentList =
-                        {
-                            "/a",
-                            msiPath,
-                            $"TARGETDIR={msiExtractionPath}",
-                            "/QN",  //  Quiet: No UI
-                        }
-                    };
 
-                    var exitCode = psi.Execute();
-                    if (exitCode != 0)
+
+                    _ = WindowsInstaller.SetInternalUI(InstallUILevel.None);
```

Comment:
Did this work before without creating the directory first when you used MSIEXEC? Admin installs usually require having the TARGETDIR created before doing the install.